### DISCUSSION TITLE
Feature request - power cable sole source of power?

### DISCUSSION THREAD

**Discussion by scm1144** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#443] [Ideas]
I've come up empty on if this is possible or not but since this app is focused on battery heath you guys may know if it is or not. My old MacBook Pro has a removable battery so I am able to use it without the battery installed by just having the power cable plugged in. Is it possible to "remove" the battery from being discharged or charged and have the laptop use the power cable for its sole power provider? This would be great for when you'd normally being hitting the battery hard like playing games. Obviously this would need monitor for the power cable being plugged in and if removed resort back to the battery.

Is this possible?

**Comment by MatthiasKerbl** [comment]
> I've come up empty on if this is possible or not but since this app is focused on battery heath you guys may know if it is or not. My old MacBook Pro has a removable battery so I am able to use it without the battery installed by just having the power cable plugged in. Is it possible to "remove" the battery from being discharged or charged and have the laptop use the power cable for its sole power provider? This would be great for when you'd normally being hitting the battery hard like playing games. Obviously this would need monitor for the power cable being plugged in and if removed resort back to the battery.
> 
> Is this possible?

Hi @bradbbh,

I am not sure which MacBooks you are talking about but with newer MacBooks (take a look at our ReadMe) and AlDente, it is possible to pause charging at a certain percentage and solely use the power adapter as a power source.

Cheers,
Matthias

**Reply by scm1144** [reply]
So this is how the Charge Limiter feature works then correct? Quoting the ReadMe, "If the MacBook has reached the desired charge limit, charging is paused and only power from the AC adapter is used." Would this be where the Pro feature Sailing comes in? In that description on your website it indicates "your MacBooks battery will always charge and discharge slightly to stay at a desired SoC, even when the Charge Limiter is activated." So by using Sailing, I can have my MacBook Pro M1 Pro charge to 80% and then with the power adapter plugged in solely use the power adapter and allows the battery to discharge very very slowly (because it is barely being used) to say 20%, where Al Dente would then allow the power adapter to charge the battery? This would be great if it worked that way.

**Reply by MatthiasKerbl** [reply]
Yes, that’s exactly how the Charge Limiter and the Sailing Mode work. However, Sailing to 20% will take forever, since your battery is not actively used as a power source while Sailing Mode is activated. That’s why I usually use a Sailing Mode range of 5%-10%.

Cheers,
Matthias

**Reply by scm1144** [reply]
Awesome, thanks for the clarification!
